### DISCUSSION TITLE
Default TimedEvent.Time value

### DISCUSSION THREAD

**Discussion by Jenix-Park** [melanchall/drywetmidi#195] [Q&A] [answered]
When the time division isn't passed via the constructor, it looks like the default value of `TimedEvent.Time` is 0.
Is this always the case?

**Comment by melanchall** [comment, marked as answer]
If you mean the second parameter of the `TimedEvent`'s constructor, it's not a time division, it's the time of an event. It's just a simple case. Just one of the constructors. There is nothing special in value of `0` :)

**Reply by Jenix-Park** [reply]
Yeah, I should've said "time", and what I was asking was if it's okay to omit it instead of `new TimedEvent(someEvent, 0)`.

**Reply by melanchall** [reply]
Yes, if you want to create an event with zero time, you can omit the second parameter.
